Source/i_system.c

```diff
@@ -257,7 +257,7 @@ void I_Quit (void)
       G_CheckDemoStatus();
    M_SaveDefaults();
 
-#if defined(MY_SDL_VER)
+#if defined(MY_SDL_VER) && defined(_MSC_VER)

```

Comment:
When I wrote this patch, the `getch()` function was not available to the MinGW toolchain -- which in turn defines `_WIN32`, so this would be the wrong check.

So, we could either leave this check as it is, because it is correct. Or we introduce a check for the `getch()` function to the `configure` script and check for `(defined(_MSC_VER) || defined(HAVE_GETCH))`. Or we replace the call to `getch()` by `getchar()` and replace `any key` by `Enter` in the printed text.